modules/core/src/v2/coins/xlm.ts

```diff
@@ -295,6 +297,40 @@ export class Xlm extends BaseCoin {
     return true;
   }
 
+  /**
+   * Create instance of stellar.MuxedAccount from M address
+   * See: https://developers.stellar.org/docs/glossary/muxed-accounts
+   *
+   * @param address
+   */
+  getMuxedAccount(address): stellar.MuxedAccount {
+    try {
+      return stellar.MuxedAccount.fromAddress(address, '0');
```

Comment:
do we need to check that `address` starts with "M"?

Reply:
the call will throw if the address is for any reason the input is not a valid muxed address